### PR TITLE
HoS's Shades

### PR DESCRIPTION
**Pull request by ps3moira**
## About the PR

Added HoS's own unique shades that has a security hud built into it.

## Why / Balance

I thought it was a bit silly HoS has to swap out and carry a sec hud and sec shades.
## Media
Screenshot 2023-12-03 205932

- [X] I have added screenshots/videos to this PR showcasing its changes ingame

**Changelog**
:cl:
- add: Added Head of Security's own shades.

### CODE REVIEW

**Comment by Colin-Tel** [comment]
Shouldn't this PR make the HoS/their locker spawn with these shades? Currently this just adds them to be mapped in.

**Comment by ps3moira** [comment]
> Shouldn't this PR make the HoS/their locker spawn with these shades? Currently this just adds them to be mapped in.

I was going to make a separate PR for this to update a few locker's missing items.

File: Resources/Prototypes/DeltaV/Entities/Clothing/Eyes/glasses.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  - type: FlashImmunity
+  - type: EyeProtection
+    protectionTime: 5
+  - type: ShowSecurityIcons
+
+  - type: Tag
+    tags:
+    - HamsterWearable
+    - WhitelistChameleon
+    - SecDogWearable # DeltaV - let Laika wear secglasses
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Entities/Clothing/Eyes/glasses.yml]: dont think this is needed since its in deltav folder

Comment (ps3moira) [reply Resources/Prototypes/DeltaV/Entities/Clothing/Eyes/glasses.yml]: True. Removed it.

**Comment by ps3moira** [comment]
> Shouldn't this PR make the HoS/their locker spawn with these shades? Currently this just adds them to be mapped in.

I decided to just add it into the locker's anyway lol. I also removed the regular sunglasses for these.

**Comment by VMSolidus** [comment]
I'm going to close this for the same reasons as #626

**Pull request closed without merging**